### DISCUSSION TITLE
Mockingbird

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1219] [Team Posts]
Sorry for the delay in the 'border resizing' fixes. I'll have something soon for you to try.

Meantime, as we were speaking of score editors, I thought you might get a kick out of this.
Reading an old BYTE magazine from 1984, there is an article about Mockingbird -  an
 advanced music notation system at Xerox PARC unlike anything that was available at the time.

I wish I could show you the article but if you google it there's some stuff about it.
Impressive stuff for its day. Here is a video about it:
https://www.youtube.com/watch?v=p45NXD_iPPI

**Comment by kybos** [comment]
Very impressive.
The origins of SCORE (the first music engraving software I ever used, and probably the only one available at that time for PC) are still much older:
https://en.wikipedia.org/wiki/SCORE_%28software%29

**Comment by terminator356** [comment]
Sad that the author passed away without leaving a continuing path for the software.
Now no one can legally obtain it.

**Comment by terminator356** [comment]
Whoa. In case you have not followed muse score lately, in the "six-degrees-of-separation" category:

I've just watched a series of youtube videos by tantacrul, the head of muse score and audacity now.
He published videos critiquing sibelius and muse score. The good, bad, and ugly.

So muse score hired him !

MusE is mentioned in that muse score video. Some history about Werner is discussed.

Now he published smart detailed videos on what he's doing with muse score.
Relevant to us: Check out this one where he has created a new **free music font** for muse score
 inspired by...you guessed it, SCORE.
He gives details about SCORE, and worked with a veteran user of it to create the font.
https://www.youtube.com/watch?v=XGo4PJd1lng

He has a lot of experience in design.
The person who has contacted us sure ain't tantacrul, I think.

**Comment by kybos** [comment]
Yes, I read about it.
Interesting video about the new font and SCORE (and MusE...).
This tantacrul surely is a talented person, it was a clever feat from MuseScore to hire him. I am sure Sibelius, Finale and Dorico/Steinberg people are not very happy about it...
Although I found his original video about MuseScore pretty arrogant and stupid in parts (ranting about the icons etc.), obviously he did not know then what Open source software is. The time would have been better spent designing new icons if he felt they are bad and could make them better, I guess that's the principle of FOSS...
